jimin/prompt.py

```
@@ -0,0 +1,156 @@
+client = OpenAI(api_key=OPENAI_API_KEY)
+MODEL = "gpt-4o-mini"
+random.seed(42); np.random.seed(42)
+
+
+# In[2]:
+
+
+def chat(messages, **kwargs):
+    params = dict(model=MODEL, temperature=kwargs.get("temperature", 0.7))
+    params["messages"] = messages
+    if "top_p" in kwargs: params["top_p"] = kwargs["top_p"]
```

Ruff (0.13.1)

Multiple statements on one line (colon)

(E701)